### PR TITLE
feat(changelog): kubernetes-changed-kapsule-and-kosmos-clusters 2023-09-07

### PR DESCRIPTION
**Pull request by ofranc**
Reporter: tgenaitay

This PR is a new changelog contribution. It has been created automatically.

Make sure to review it carefully before merging it.

### CODE REVIEW

File: changelog/september2023/2023-09-07-kubernetes-changed-kapsule-and-kosmos-clusters.mdx [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+title: Kapsule and Kosmos clusters have a new limit on first offer: 150 nodes maximum
```
Comment (nerda-codes) [inline changelog/september2023/2023-09-07-kubernetes-changed-kapsule-and-kosmos-clusters.mdx:2]:
```suggestion
title: Kapsule and Kosmos clusters have a new limit on first offer.
```

File: changelog/september2023/2023-09-07-kubernetes-changed-kapsule-and-kosmos-clusters.mdx [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+status: changed
+author:
+    fullname: 'Join the #k8s channel on Slack.'
+    url: 'https://slack.scaleway.com'
+date: 2023-09-07
+category: containers
+product: kubernetes
+---
+
+Effective September 1st 2023. To go beyond 150 nodes, Scaleway offers a range of dedicated control planes to cater to a wide range of use-cases. Refer to our [Kubernetes control plane offers overview](https://www.scaleway.com/en/docs/containers/kubernetes/reference-content/kubernetes-control-plane-offers/) for more information.
```
Comment (nerda-codes) [inline changelog/september2023/2023-09-07-kubernetes-changed-kapsule-and-kosmos-clusters.mdx:12]:
```suggestion
Starting September 1st 2023, Kapsule and Kosmos clusters are limited to 150 nodes maximum on first offer. Scaleway offers a range of dedicated control planes to cater to a wide range of use-cases, to go beyond 150 nodes. Refer to our [Kubernetes control plane offers overview](/containers/kubernetes/reference-content/kubernetes-control-plane-offers/) for more information.
```

Comment (ldecarvalho-doc) [reply changelog/september2023/2023-09-07-kubernetes-changed-kapsule-and-kosmos-clusters.mdx:12]: I don't know if this is what was meant. @tgenaitay can you clear it up for us? 🙏🏻

Comment (tgenaitay) [reply changelog/september2023/2023-09-07-kubernetes-changed-kapsule-and-kosmos-clusters.mdx:12]: @ldecarvalho-doc
As a non native speaker, it sounds like the same to me. The point is: to go beyond the new limit, DCP are here.

Comment (tgenaitay) [reply changelog/september2023/2023-09-07-kubernetes-changed-kapsule-and-kosmos-clusters.mdx:12]: LGTM!

**Review by RoRoJ — APPROVED** [review]

**Review by nerda-codes — APPROVED** [review]

**Pull request merged**